MediaRecorder.bs

```diff
@@ -286,11 +276,35 @@ interface MediaRecorder : EventTarget {
     <li>Set |recorder|'s {{state}} to {{recording}}, and run the following steps
     in parallel:
     <ol>
-      <li>Start recording all tracks in |tracks| using the {{MediaRecorder}}'s
-      current configuration, and gather the data into a {{Blob}} <var>blob</var>
-      and queue a task, using the DOM manipulation task source, to
-      <a>fire an event</a> named <a>start</a> at <var>recorder</var>, then return
-      <code>undefined</code>.</li>
+      <li>Let the User Agent specify the |recorder|'s configuration for
+      recording all the tracks in |tracks|, given its current constraints. If
```

Comment:
There's already a current configuration that has been constrained above. "current constraints" would be a new concept, and "specify the recorder's configuration" seems to overwrite the current configuration.

Instead of this paragraph, perhaps "Start recording all tracks in |tracks| using the recorder's current configuration" can be amended to mention that there might be room to choose container and codec still?

Reply:
I rephrased it, does this work better?